src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -219,15 +219,31 @@ public static bool IsConsumptionSku(this IEnvironment environment)
         /// <returns><see cref="true"/> if running in the FlexConsumption Sku, false otherwise.</returns>
         public static bool IsFlexConsumptionSku(this IEnvironment environment)
         {
-            string value = environment.GetEnvironmentVariable(AzureWebsiteSku);
-            if (string.Equals(value, ScriptConstants.FlexConsumptionSku, StringComparison.OrdinalIgnoreCase))
+            string sku = environment.GetEnvironmentVariable(WebsiteSkuName);
```

Comment:
I think we should leave this as it was, based on WEBSITE_SKU, and using IsLinuxConsumptionOnLegion as a backup check. Why isn't that sufficient?